package.json

```diff
@@ -24,14 +31,26 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "istanbul": "^0.3.2",
-    "jshint": "^2.5.10",
-    "mocha": "^2.0.1",
-    "should": "^4.1.0"
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.2.220",
+    "@swc/jest": "^0.2.22",
+    "@types/debug": "^4.1.7",
+    "@types/jest": "^28.1.6",
```

Comment:
I _think_ this is no longer needed, since `jest` provide their own types. Can you try to remove this to confirm?

Reply:
Didn't work